css/tokens/index.js

```diff
@@ -32,7 +32,7 @@ async function createTokens() {
 		await fs.ensureDir(outfolder);
 		await Promise.all([
 			fs.writeJSON(`${outfileStem}.json`, collection),
-			quicktypeJSON('AtlasTokens', JSON.stringify(collection), `${outfileStem}.d.ts`, 'typescript')
+			quicktypeJSON('AtlasTokens', JSON.stringify(collection), `${outfileStem}.ts`, 'typescript')
 		]);
 		console.log(`Tokens written to "${path.join(process.cwd(), '/dist/index.json')}".`);
```

Comment:
Doesn't have to be addressed in this PR: does this actually write to `'/dist/index.json'`? Should it be `'/dist/token.json'`?

Reply:
The outfile stem is `/dist/tokens` and so these two lines output `/dist/tokens.ts` for (non-ambient [not d.ts]) types and `/dist/tokens.json` for the tokens themselves.